### DISCUSSION TITLE
Any chance to integrate GLIGEN?

### DISCUSSION THREAD

**Discussion by OedoSoldier** [Mikubill/sd-webui-controlnet#519] [Ideas]
Demo: https://huggingface.co/spaces/gligen/demo
GitHub: https://github.com/gligen/GLIGEN

This thing is so cool!

**Comment by Mikubill** [comment]
Great. We are waiting for paint-with-words impl on branch
https://github.com/lwchen6309/sd-webui-controlnet, once they finished we
will add gligen support immediately.

**Reply by OedoSoldier** [reply]
Yeeeeees

**Reply by catboxanon** [reply]
Private repo? Either way, looking forward to GLIGEN support!

**Reply by scruffynerf** [reply]
> https://github.com/lwchen6309/sd-webui-controlnet

https://github.com/lwchen6309/paint-with-words-sd

**Reply by lifang1996** [reply]
Hi @Mikubill  https://github.com/lwchen6309/paint-with-words-sd seems to have been done. Will it be added to support these days？

**Reply by OedoSoldier** [reply]
Hi @Mikubill, it seems [paint with words](https://github.com/lwchen6309/sd-webui-controlnet-pww/tree/main) is ready, is there a plan to merge it into this project?

**Reply by kowalgregy** [reply]
Looking forward to GliGen support.

**Reply by AoyuQC** [reply]
@Mikubill , any plan to integrate GliGen? It's a really useful tool.

**Reply by xdejiko** [reply]
Greeting @Mikubill  is it still pending to be integrate it? it's a awesome feature.
